src/test/java/org/springframework/data/jpa/repository/AbstractPersistableIntegrationTests.java

```diff
@@ -66,4 +68,16 @@ void equalsWorksForProxiedEntities() {
 
 		assertThat(proxy).isEqualTo(proxy);
 	}
+
+	@Test // DATAJPA-1377
```

Comment:
These should note the gh issue number. Apparently the DATAJPA are the old Jira identifiers